src/style.css

```diff
@@ -1 +1,77 @@
-/* Styles go here */
+html {
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  --item-color: #00acdc;
+  --nav-size: 60px;
+}
+
+body {
+  display: block;
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0;
```

Comment:
You can reset default margin/padding once for all elements that need, for example
_body, 
header, 
ul,
p {
  margin: 0;
  padding: 0;
}_
Please check for all cases where you reset default styles